app/views/admin_posts/_admin_post.html.erb

```diff
@@ -0,0 +1,39 @@
+<% # expects "admin_post" %>
+<div class="header">
+  <h3 class="heading">
+    <%= link_to admin_post.title.html_safe, admin_post %>
+  </h3>
+</div>
+<h4 class="landmark heading"><%= ts("Post Header") %></h4>
+<div class="wrapper">
+  <dl class="meta">
+    <dt class="published"><%= ts("Published:") %></dt>
+    <dd class="published"><%= admin_post.created_at %></dd>
+    <% if admin_post.translated_post %>
+      <dt class="original translations"><%= ts("Original:") %></dt>
+      <dd class="original translations"><%= link_to admin_post.translated_post.title, admin_post.translated_post %></dd>
+    <% elsif !admin_post.translations.empty? %>
+      <dt class="translations"><%= ts("Translations:") %></dt>
+      <dd class="translations">
+        <ul class="languages commas">
+          <% for translation in admin_post.translations %>
+            <li><%= link_to translation.language.name, translation %></li>
+          <% end %>
+        </ul>
+      </dd>
+    <% end %>
+    <% if admin_post.tags.length > 0 %>
```

Comment:
The tags used to be just sitting unlabeled under the contents of the post. Now they're up here in the newly-added meta.